apps/web/src/pages/AnnouncementsPage/labels.ts

```diff
@@ -0,0 +1,53 @@
+import { defineMessage } from "react-intl";
+
+const labels = {
+  isEnabled: defineMessage({
+    defaultMessage: "Enabled",
+    id: "QpghXO",
+    description: "A switch to enable or disable a setting",
+  }),
+  publishDateUtc: defineMessage({
+    defaultMessage: "Publish date (UTC)",
+    id: "5cll86",
+    description:
+      "A date at which data will be published, in the UTC time standard",
+  }),
+  publishDateLocal: defineMessage({
+    defaultMessage: "Publish date (Local)",
+    id: "DQ8q0K",
+    description:
+      "A date at which data will be published, in the local time zone",
+  }),
+  expiryDateUtc: defineMessage({
+    defaultMessage: "Expiry date (UTC)",
+    id: "j9zYRY",
+    description: "A date at which data will expire, in the UTC time standard",
+  }),
+  expiryDateLocal: defineMessage({
+    defaultMessage: "Expiry date (Local)",
+    id: "KH3yb6",
+    description: "A date at which data will expire, in the local time zone",
+  }),
+  titleEn: defineMessage({
+    defaultMessage: "English - Title",
```

Comment:
The format we used on other parts of the site for English and French versions is: `{fieldName} ({languageName})`.

Reply:
Hmm, we actually use both.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/cc39dd46-86d5-472b-8ece-98423bc6160f)
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/cac64765-b3da-4f6b-a436-064eec39876f)

The Edit Pool page is much newer so I assumed that was the preferred form.  @Jerryescandon which standard are we going with?

Reply:
Oh, previous one, I will update the figma to match.

Reply:
OK, I've updated it to use the parenthetical form.
[copy tweaks](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9387/commits/b72b30d8df33d7acf089a0fa2b26ae6fbc8f2b83)